### PR TITLE
Online emissions (dust and sea salt) code changes after implementing EAMxx interface codes

### PR DESCRIPTION
**Pull request by singhbalwinder**

### CODE REVIEW

**Review by mjs271 — APPROVED** [review]
Looks great--thanks for ironing out those interface decisions!

File: src/mam4xx/aero_model_emissions.hpp [hunk 1]
```diff
@@ -910,62 +914,74 @@ void aero_model_emissions(
...
+  // Populate online emission data structure with inputs
   OnlineEmissionsData online_emiss_data;
+  for (int idst = 0; idst < dust_nflux_in; ++idst) {
+    online_emiss_data.dust_flux_in[idst] = dstflx(idst);
+  }
+  online_emiss_data.surface_temp = sst;
+  online_emiss_data.u_bottom = u_bottom;
+  online_emiss_data.v_bottom = v_bottom;
+  online_emiss_data.ocean_frac = ocnfrac;
+  // FIXME: HARDWIRED- remove it!!!!
```
Comment (odiazib) [inline src/mam4xx/aero_model_emissions.hpp:968]: Is this FIXME still valid?

**Review by odiazib — APPROVED** [review]

File: src/mam4xx/aero_model_emissions.hpp (hunk 1 above)
Comment (singhbalwinder) [reply src/mam4xx/aero_model_emissions.hpp:968]: No, I have removed the hardwired code. I will remove this comment.

File: src/mam4xx/aero_model_emissions.hpp [hunk 2]
```diff
@@ -910,62 +947,79 @@ void aero_model_emissions(
...
+  init_dust_dmt_vwr(dust_data);
+
+  // FIXME: why not send it as an input as a view????
   Real cflux[pcnst];
   for (int i = 0; i < pcnst; ++i) {
     cflux[i] = cflux_(i);
   }
-  aero_model_emissions(online_emiss_data, seasalt_data, dust_data, cflux);
+  aero_model_emissions(online_emiss_data, seasalt_data, dust_data,
+                       cflux); // out
```
Comment (overfelt) [inline src/mam4xx/aero_model_emissions.hpp:1023]: @singhbalwinder : Do you need to copy the local value of cflux to the cflux_ Kokkos view after the call to aero_model_meeisions?

**Review by overfelt — APPROVED** [review]

**Pull request merged**